src/Raven.Studio/typescript/components/pages/database/settings/documentSchema/partials/DocumentSchemaSelectActions.tsx

```diff
@@ -42,8 +46,8 @@ export default function DocumentSchemaSelectActions() {
         setTrue: setIsTogglingGlobalStatusTrue,
         setFalse: setIsTogglingGlobalStatusFalse,
     } = useBoolean(false);
-    const [operationConfirm, setOperationConfirm] = React.useState<OperationConfirm>(null);
     const confirm = useConfirm();
+    const [operationConfirm, setOperationConfirm] = React.useState<OperationConfirm>(null);
```

Comment:
We alawys omit `React.` part in this case
```suggestion
    const [operationConfirm, setOperationConfirm] = useState<OperationConfirm>(null);
```